packages/playwright-dashboard/src/ghaMarkdownReporter.ts

```diff
@@ -91,13 +89,27 @@ class GHAMarkdownReporter extends MarkdownReporter {
     return prId;
   }
 
+  private _magicComment() {
+    return `<!-- Generated by Playwright markdown reporter for ${this._workflowRunName()} -->`;
```

Comment:
We should probably include job name as well, in case there are several reports generated by  different jobs in the same workflow.